src/data/bool/basic.lean

```diff
@@ -132,6 +132,9 @@ theorem band_elim_right : ∀ {a b : bool}, a && b → b := dec_trivial
 
 @[simp] theorem bnot_true : bnot tt = ff := rfl
 
+@[simp] theorem ne_bnot : ∀ {a b : bool}, ¬a = !b ↔ a = b := dec_trivial
+@[simp] theorem bnot_ne : ∀ {a b : bool}, ¬!a = b ↔ a = b := dec_trivial
```

Comment:
Should we simplify `a = !b` to `a ≠ b` instead?

Reply:
I feel like the equality is more useful because you can substitute it

Reply:
Then the names should use `not_eq` instead of `ne`: IMHO, `ne` is for `≠` (should we have non-simp versions with `ne`?).